mergify_engine/rules/parser.py

```diff
@@ -290,6 +292,10 @@ def parse(v: str) -> typing.Any:
 
     position = _skip_ws(v, length, position + len(attribute))
 
+    if not quantity and attribute in ATTRIBUTES_WITH_ONLY_LENGTH:
+        raise ConditionParsingError(
+            f"`#` modifier is required for attribute: `{attribute}`"
```

Comment:
```suggestion
            f"`#` operator is required for attribute: `{attribute}`"
```

Reply:
Hum, use used modifier everywhere. Operator is reserved for > < = ...